windows/draw.hpp

```diff
@@ -49,6 +49,10 @@ class textDrawingEffect : public IUnknown {
 		this->hasUnderlineColor = false;
 	}
 
+    virtual ~textDrawingEffect()
+    {
+    }
+
```

Comment:
Tabs needed here I guess :)

Reply:
Doh. Bane of my life these days. Switching through so many different projects and platforms right now. At least I remembered to use `{` and `}` instead of `begin` and `end.` :-)